src/components/Tour/Tour.scss

```diff
@@ -1,7 +1,7 @@
 @use '../../../node_modules/intro.js/introjs.css' as *;
```

Comment:
do we need the relative path here for the intro.js css ?

Reply:
Probably not, but I kept it mainly because I didn't know much about the intro.js library.